### DISCUSSION TITLE
Documentation of openApi

### DISCUSSION THREAD

**Discussion by cynthiatoussi** [OAI/OpenAPI-Specification#3224] [Q&A]
Please can I have solutions of this exercise?

![Screenshot_2023-03-31-13-13-31-841_com.google.android.apps.docs.jpg](https://user-images.githubusercontent.com/117768938/229232541-5b737b79-bb27-4f16-99a2-a97e9dbe0df8.jpg)

**Comment by lornajane** [comment]
This doesn't seem relevant to the project, but more of a homework assignment.
